src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -0,0 +1,225 @@
+foreground color and background color is 4.5:1. If text size is 200%, the
+button scales with text so there is no overlap.
+
+If a tooltip is associated with a form control (text, radio, checkbox, etc),
+the text for the tooltip should be included in the form control label.
+
+Resources:
+
+- [W3C WAI ARIA Tooltip](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)
+- [MDN ARIA: tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)
+- [Chakra UI Tooltip](https://chakra-ui.com/docs/components/overlay/tooltip)
+
+## Examples
+
+### Tooltips on form options
+
+When using the `Tooltip` component in forms containing `Checkbox`, `Radio`, `Toggle`.
+
+<Canvas>
+  <DSProvider>
+    <RadioGroup
+      labelText="What would you like to request?"
+      id="radio-group"
+      name="tooltip-example"
+    >
+      <Tooltip
+        content="General library orientation or a customized research lesson"
+        shouldWrapChildren
+      >
+        <Radio id="radio1" labelText="Class Visit" value="Class Visit" />
+      </Tooltip>
+      <Tooltip
+        content="Guided exploration of the library and its resources"
+        shouldWrapChildren
+      >
+        <Radio id="radio2" labelText="Group Tour" value="Group Tour" />
+      </Tooltip>
+      <Tooltip
+        content="Invite library staff to share resources at your community event"
+        shouldWrapChildren
+      >
+        <Radio
+          id="radio3"
+          labelText="Offsite Community Outreach"
+          value="Offsite Community Outreach"
+        />
+      </Tooltip>
+    </RadioGroup>
+  </DSProvider>
+</Canvas>
+
+### Tooltip on Linked Icon
+
+<Canvas>
+  <DSProvider>
+    <Tooltip content="This is the tooltip text">
+      <Link type="action" href="#passed-in-link">
+        <Icon decorative id="icon-id" name="search" size="xxlarge" />
+      </Link>
+    </Tooltip>
+  </DSProvider>
+</Canvas>
+
+### Tooltip with Icon Content
+
+<Canvas>
+  <DSProvider>
+    <Tooltip
+      content={
+        <Icon
+          decorative
+          id="icon-id"
+          name="search"
+          size="xxlarge"
+          color="ui.white"
+        />
+      }
+    >
+      Hover to see search icon
+    </Tooltip>
+  </DSProvider>
+</Canvas>
+
+### Tooltip with Image Content
+
+<Canvas>
+  <DSProvider>
+    <Tooltip
+      content={<Image src="https://placeimg.com/300/300/animals?x=1" alt="" />}
+    >
+      Hover to see image
+    </Tooltip>
+  </DSProvider>
+</Canvas>
+
+### Disabling Tooltip
+
+A tooltip can be disabled by passing the `isDisabled` prop.
+
+<Canvas>
+  <DSProvider>
+    <Tooltip
+      content="This is a tooltip with information describing the hovered object."
+      isDisabled
+    >
+      <Button id="tooltip-button">Hover me</Button>
+    </Tooltip>
+  </DSProvider>
+</Canvas>
+
+By default, the `Tooltip` is not shown for a disabled `Button`. To show the `Tooltip`
```

Comment:
This is a great example. Is this something that Chakra is doing by default and so our own components behave the same way?

Reply:
Yes this is due to Chakra's default behavior: https://chakra-ui.com/docs/components/tooltip/usage#disabling-tooltip